### ISSUE TITLE
[BUG] Paging (last, before) will return wrong value

### ISSUE THREAD

**Issue by zerox12311** [doug-martin/nestjs-query#281]
In my test case, I have 10 rows data in my database.
With default **first:10** parameter
RETURN:
```json
{
    "data": {
        "permissions": {
            "pageInfo": {
                "startCursor": "YXJyYXljb25uZWN0aW9uOjA=",
                "endCursor": "YXJyYXljb25uZWN0aW9uOjk=",
                "hasPreviousPage": false,
                "hasNextPage": true
            },
            "edges": [
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
                    "node": {
                        "id": 1
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
                    "node": {
                        "id": 2
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
                    "node": {
                        "id": 3
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
                    "node": {
                        "id": 4
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
                    "node": {
                        "id": 5
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
                    "node": {
                        "id": 6
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjY=",
                    "node": {
                        "id": 7
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjc=",
                    "node": {
                        "id": 8
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjg=",
                    "node": {
                        "id": 9
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjk=",
                    "node": {
                        "id": 10
                    }
                }
            ]
        }
    }
}
```
---

And I use **first:3** parameter
RETURN:
```json
{
    "data": {
        "permissions": {
            "pageInfo": {
                "startCursor": "YXJyYXljb25uZWN0aW9uOjA=",
                "endCursor": "YXJyYXljb25uZWN0aW9uOjI=",
                "hasPreviousPage": false,
                "hasNextPage": true
            },
            "edges": [
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
                    "node": {
                        "id": 1
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
                    "node": {
                        "id": 2
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
                    "node": {
                        "id": 3
                    }
                }
            ]
        }
    }
}
```
---

And use (first, after) parameter, it worked fine!
(first: 3, after:"YXJyYXljb25uZWN0aW9uOjI=")
// first:3, after: id 3
RETURN:
```json
{
    "data": {
        "permissions": {
            "pageInfo": {
                "startCursor": "YXJyYXljb25uZWN0aW9uOjM=",
                "endCursor": "YXJyYXljb25uZWN0aW9uOjU=",
                "hasPreviousPage": true,
                "hasNextPage": true
            },
            "edges": [
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
                    "node": {
                        "id": 4
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
                    "node": {
                        "id": 5
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
                    "node": {
                        "id": 6
                    }
                }
            ]
        }
    }
}
```

---

But this time, I use (last, before)
(last:2, before:"YXJyYXljb25uZWN0aW9uOjM=")
// last:2, before: id 4
It **should be** return id 2 and id 3
But it return:
```json
{
    "data": {
        "permissions": {
            "pageInfo": {
                "startCursor": "YXJyYXljb25uZWN0aW9uOjE=",
                "endCursor": "YXJyYXljb25uZWN0aW9uOjI=",
                "hasPreviousPage": false,
                "hasNextPage": true
            },
            "edges": [
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
                    "node": {
                        "id": 1
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
                    "node": {
                        "id": 2
                    }
                }
            ]
        }
    }
}
```

I think it is a bug

**Comment by doug-martin** [comment]
Fixed under `v0.13.1`. Thank you for the detailed example!

**Comment by thehappycoder** [comment]
In my case, if I use the same example, the last query would return:
```
{
                    "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
                    "node": {
                        "id": 3
                    }
                },
                {
                    "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
                    "node": {
                        "id": 4
                    }
                },
```
